recipes/opacus/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "opacus" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opacus-{{ version }}.tar.gz
+    sha256: c84fcbebe01bb185d3d2abcad135c47687afa93a0c7f6fd92d0018934b039cb9
+  - url: https://raw.githubusercontent.com/pytorch/opacus/main/dev_requirements.txt
```

Comment:
Why is this needed? 🤔

Reply:
The setup required the `dev_requirements` file, for some reason. The recipe would not build without this.

Reply:
Improved this to use `https://raw.githubusercontent.com/pytorch/opacus/v{{ version }}/dev_requirements.txt`.

Reply:
That can be fine temporarily, but the issue about it missing needs to be opened with upstream.

Reply:
Agreed, I'm going to put a PR once this is merged.